pkg/cmd/cli/cmd/newapp.go

```diff
@@ -845,3 +849,14 @@ func (r *configSecretRetriever) CACert() (string, error) {
 	}
 	return "", nil
 }
+
+func checkGitInstalled(w io.Writer) {
+	gitBinary := "git"
+	if goruntime.GOOS == "windows" {
+		gitBinary = "git.exe"
+	}
+	_, err := exec.LookPath(gitBinary)
+	if err != nil {
+		fmt.Fprintf(w, "warning: Cannot find git. Ensure that it is installed and in your path.\n")
```

Comment:
It matches the format of other CLI warnings/errors.
warning: blah
error: blah

However, it doesn't match the warning we include when doing a binary build because that's part of the output of a successful run. With git, I think we want to show the warning regardless.